server/apps/preferences.py

```diff
@@ -120,6 +120,13 @@ class PreferencesResource(Resource):
         'category': 'dateline'
     })
 
+    superdesk.register_default_user_preference('categories:preferred', {
```

Comment:
Some behave test would be nice as well, at least to get a feeling how this new preference is supposed to work form API's point of view.